keras/utils.py

```diff
@@ -14,16 +15,23 @@
 from tanda.tan import PretrainedTAN
 from tanda.transformer import PadCropTransformer
 
-
-def load_pretrained_tan(config_path, checkpoint_path, tfs, dims=[32, 32, 3]):
-    # Load config
-    with open(config_path, 'r') as f:
+def load_pretrained_tan(path):
+    # Load config dictionary from run log
+    with open(os.path.join(path, 'run_log.json'), 'r') as f:
```

Comment:
Is this where this file is always located? I think it's usually in `logs`, just like this for the pretrained dir

Reply:
Fixed